src/main/java/pl/gov/coi/pomocua/ads/accomodations/AccommodationOfferDefinitionDTO.java

```diff
@@ -50,6 +51,7 @@ public void applyTo(AccommodationOffer offer) {
         offer.guests = guests;
         offer.lengthOfStay = lengthOfStay;
         offer.hostLanguage = new LinkedList<>(hostLanguage);
-        offer.phoneNumber = Optional.ofNullable(phoneNumber).map(PhoneUtil::normalize).orElse(null);
+        offer.phoneNumber = PhoneUtil.getPhoneDetails(phoneNumber).map(PhoneDetails::nationalNumber).orElse(null);
+        offer.phoneCountryCode = PhoneUtil.getPhoneDetails(phoneNumber).map(PhoneDetails::countryCode).orElse(null);
```

Comment:
Maybe `PhoneUtil.getPhoneDetails(...)` could be done just once to set both of the fields?
Like:
```
PhoneUtil.getPhoneDetails(...).ifPresent(phoneDetails -> {
  offer.phoneNumber = phoneDetails.nationalNumber();
  offer.countryCode = phoneDetails.countryCode();
});
```